logserver/src/test/java/com/yahoo/logserver/handlers/archive/ArchiverHandlerTestCase.java

```diff
@@ -203,13 +203,20 @@ public void testRotation() throws IOException {
 
     @Test
     public void testCacheEldestEntry() throws IOException {
+        try {
+
         LogWriterLRUCache cache = new LogWriterLRUCache(5, (float) 0.75);
         String d = "target/tmp/logarchive";
         FilesArchived archive = new FilesArchived(new File(d));
         for (int i = 0; i < cache.maxEntries + 10; i++) {
             cache.put(i, new LogWriter(d+"/2018/12/31/17", 5, archive));
         }
         assertEquals(cache.size(), cache.maxEntries);
+
+        } catch (NullPointerException e) {
+            System.err.println("null: "+e);
+            e.printStackTrace();
```

Comment:
This looks like debug code that was accidentally included in the commit.

Reply:
debug code removed.